<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777213</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="status">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <attribute name="title">
     <string>Current Status</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>960</width>
       <height>450</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLCDNumber" name="road_temp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>141</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>현재온도</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>141</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>현재습도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="road_humidity">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>141</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>대기온도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="air_temp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>설정온도</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>설정습도</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>170</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>설정대기온도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="heat_road_temp_2">
      <property name="geometry">
       <rect>
        <x>739</x>
        <y>30</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="set_road_humidity_2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="set_air_temp_2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>170</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_pre_heat_on">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>330</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>예열 가동 상태 (Channel1)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_heat_on">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>본가동 상태 (Channel1+2)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <attribute name="icon">
     <iconset>
      <normaloff>../../../peltier/peltier_sw/pi_fw/0630_1/image/parameter_on.png</normaloff>../../../peltier/peltier_sw/pi_fw/0630_1/image/parameter_on.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>960</width>
       <height>450</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLCDNumber" name="pre_heat_road_temp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>5.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>예열 설정 노면 온도</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>251</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>본 가동 설정 노면 온도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="heat_road_temp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>191</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>설정 노면 습도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="set_road_humidity">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>3.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>241</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>예열 작동 지속 시간</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>231</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>본 작동 지속 시간</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>251</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>자동 Heating ON/OFF</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="pre_heat_on_time">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>30.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="heat_on_time">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>100</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>30.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>191</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>설정 대기 온도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="set_air_temp">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>10.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_PRE_HEAT_ON">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>강제 예열 가동 (Channel1)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_HEAT_ON">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>강제 본 가동 (Channel1+2)</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="auto_heat_on_off">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>170</y>
        <width>181</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>5.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_warning">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>401</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 0)</string>
      </property>
      <property name="text">
       <string>Cannot change config value
(thermal film is working!!)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="admin">
    <attribute name="icon">
     <iconset>
      <normaloff>../../../peltier/peltier_sw/pi_fw/0630_1/image/engineer_on.png</normaloff>../../../peltier/peltier_sw/pi_fw/0630_1/image/engineer_on.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Admin</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>960</width>
       <height>450</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LOG</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
